Update imported filtering capabilities (#509)
plausible/docs: docs/stats-api.md

#### Filtering imported stats

Filtering by imported data is limited. The general rule is that you cannot filter by two different properties at the same time.
For example, `filters=event:page==/;visit:source==Twitter` is not able to return any imported results. The same happens when you try to filter by one property and break down by another.

> There are some exceptions though, the following properties are aggregated and grouped together, which means you can use them together in a query as well.

Reviewer: ```suggestion
There are some exceptions though. The following properties are aggregated and grouped together and can be combined in a query:
```
